src/main/java/edu/harvard/iq/dataverse/pidproviders/FakePidProviderServiceBean.java

```diff
@@ -0,0 +1,66 @@
+package edu.harvard.iq.dataverse.pidproviders;
+
+import edu.harvard.iq.dataverse.AbstractGlobalIdServiceBean;
+import edu.harvard.iq.dataverse.DvObject;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.ejb.Stateless;
+
+@Stateless
+public class FakePidProviderServiceBean extends AbstractGlobalIdServiceBean {
+
+    @Override
+    public boolean alreadyExists(DvObject dvo) throws Exception {
+        return true;
+    }
+
+    @Override
+    public boolean registerWhenPublished() {
+        return false;
+    }
+
+    @Override
+    public List<String> getProviderInformation() {
+        ArrayList<String> providerInfo = new ArrayList<>();
+        String providerName = "FAKE";
+        String providerLink = "http://dataverse.org";
```

Comment:
Could this be replaced with the dataverse FQDN/URL? That way we could generate fake, but working PID URLs pointing to the instance, couldn't we?